source/evp_override.c

```diff
@@ -1029,3 +1027,8 @@ void evp_md_ctx_shallow_free(EVP_MD_CTX *ctx) {
     free(ctx);
     // Does not free EVP_KEY, since this is always done separately in our use cases
 }
+
+void EVP_MD_CTX_set_flags(EVP_MD_CTX *ctx, int flags) {
+    assert(__CPROVER_r_ok(ctx, sizeof(*ctx)));
```

Comment:
Should this be `__CPROVER_w_ok` instead? We're modifying the `flags` field below ...

Reply:
Good catch! Thanks! Implemented as suggested.